iXBRLViewerPlugin/__init__.py

```diff
@@ -27,21 +37,15 @@
 #
 #     example uploading an ESEF report package and receiving a zip of viewable .xhtml and viewer javascript file:
 #
-#         curl -X POST "-HContent-type: application/zip" 
-#              -T /Users/mystuff/ESMA/samples/bzwbk_2016.zip 
-#              -o ~/temp/out.zip 
+#         curl -X POST "-HContent-type: application/zip"
+#              -T /Users/mystuff/ESMA/samples/bzwbk_2016.zip
+#              -o ~/temp/out.zip
 #              "http://localhost:8080/rest/xbrl/validation?&media=zip&plugins=iXBRLViewerPlugin&packages=somewhere/esef_taxonomy_2017.zip"
 #
 #     In the zip, the iXBRLViewer files are in a subdirectory VIEWER_BASENAME_SUFFIX to separate them from possible EdgarRenderer and other output files
 #
 from .iXBRLViewer import IXBRLViewerBuilder, IXBRLViewerBuilderError
```

Comment:
nit: move this up with the rest of the imports

Reply:
Pre-existing: But cando